### DISCUSSION TITLE
[Support]: Unable to delete points when editing zone

### DISCUSSION THREAD

**Discussion by MoHf7f4** [blakeblackshear/frigate#12867] [General Support] [answered]
### Describe the problem you are having

Right-click use to delete a point.  Now it opens a chrome menu.
![image](https://github.com/user-attachments/assets/76fb7c26-c019-43a7-8d0a-dd1a32a831ca)

### Steps to reproduce

1. Use Chrome
2. Settings -> Masks/Zones -> Edit a zone (the pencil)
3. Right-click on any point

### Version

0.14.0

### Frigate config file

```yaml
Feature related
```

### Relevant log output

```shell
none
```

### Operating system

Debian

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

All of them.

**Comment by hawkeye217** [comment, marked as answer]
There is a PR up to add this functionality: https://github.com/blakeblackshear/frigate/pull/12744

Use the Undo button in the sidebar to remove the last point in the polygon.
